GW2EIEvtcParser/EIData/ProfHelpers/Elementalist/EvokerHelper.cs

```diff
@@ -11,18 +12,100 @@ namespace GW2EIEvtcParser.EIData;
 
 internal static class EvokerHelper
 {
-    internal static readonly List<InstantCastFinder> InstantCastFinder = [];
+    internal static readonly List<InstantCastFinder> InstantCastFinder = 
+    [
+        new MinionCastCastFinder(IgniteSkill, IgniteDamage),
+        new MinionCastCastFinder(SplashSkill, 999999), // TODO Find skill id casted by the pet
+        new MinionCastCastFinder(ZapSkill, ZapDamage),
+        new MinionCastCastFinder(CalcifySkill, 999999), // TODO Find skill id casted by the pet
+    ];
 
-    internal static readonly IReadOnlyList<DamageModifierDescriptor> OutgoingDamageModifiers = [];
+    internal static readonly IReadOnlyList<DamageModifierDescriptor> OutgoingDamageModifiers = 
+    [
+        // Elemental Balance
+        // TODO Verify behaviour https://wiki.guildwars2.com/wiki/Elemental_Balance
+        new DamageLogDamageModifier(Mod_ElementalBalanceOutgoing5, "Elemental balance", "5% if hp <= 50%", DamageSource.All, 5.0, DamageType.Strike, DamageType.All, Source.Evoker, TraitImages.ElementalBalance, (x, log) => x.From.GetCurrentHealthPercent(log, x.Time) <= 50.0, DamageModifierMode.All),
+        new DamageLogDamageModifier(Mod_ElementalBalanceOutgoing10, "Elemental balance", "10% if hp >= 50%", DamageSource.All, 10.0, DamageType.Strike, DamageType.All, Source.Evoker, TraitImages.ElementalBalance, (x, log) => x.From.GetCurrentHealthPercent(log, x.Time) >= 50.0, DamageModifierMode.All),
```

Comment:
Same for incoming, one of the condition has to be strict, I don't think evoker gets both advantages when sitting at exactly 50%